settings.gradle

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 
 plugins {
-    id 'com.gradle.develocity' version '3.17.6'
+    id 'com.gradle.develocity' version '3.19'
     id 'com.gradle.common-custom-user-data-gradle-plugin' version '2.0.2'
 }
 
```

Comment:
Conflict was below in L25.

`trunk` change was (https://github.com/apache/kafka/commit/e9e03faf87879ad3742164938000f6abbfe617ca#diff-7f825392aa37acd1cee0c2e7b9bb7366ad6eac64f3e6cdd816e156bcb69d30de)
```
-    server = "https://ge.apache.org"
+    server = "https://develocity.apache.org"
```
But line does not exiting in our version. Rejected the change.